frontend/src/pull-card/flags.tsx

```diff
@@ -86,6 +88,13 @@ export const Flags = memo(function Flags({ pull }: { pull: Pull }) {
           icon={faSnowflake}
         />
       )}
+      {mergeConflict && (
```

Comment:
I think I'd use `pull.hasMergeConflicts()` directly here to reduce the indirection.

Reply:
I agree, resolved via https://github.com/iFixit/pulldasher/pull/356/commits/88db70109f0aa106040bd2afe84d455b2bd26df4